kibana-reports/public/components/report_definitions/edit/edit_report_definition.tsx

```diff
@@ -97,7 +97,6 @@ export function EditReportDefinition(props) {
 
   const callUpdateAPI = async (metadata) => {
     const { httpClient } = props;
-
```

Comment:
Can we remove this file from the PR?

Reply:
do you mean `edit_report_definition.tsx`

Reply:
Yeah- it looks like the only change is an empty line removal?

Reply:
done